### PR TITLE
Docs: Fix promtail <client_config> block heading

### PR DESCRIPTION
**Pull request by DylanGuedes**
**What this PR does / why we need it**:
Fix promtail <client_config> block heading, wrongly named <client>

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated
- [ ] Add an entry in the `CHANGELOG.md` about the changes.

### CODE REVIEW

File: docs/sources/clients/promtail/configuration.md [hunk 1]
```diff
@@ -157,10 +157,9 @@ The `server` block configures Promtail's behavior as an HTTP server:
 [health_check_target: <bool> | default = true]
 ```
 
-## clients
+## client_config
```
Comment (trevorwhitney) [inline docs/sources/clients/promtail/configuration.md:160]: Are we just changing the header to match the `<client_config>` referenced above? If so should we do the same for `server` -> `server_config`, and `positions` -> `position_config`?

File: docs/sources/clients/promtail/configuration.md [hunk 2]
```diff
@@ -157,10 +157,9 @@ The `server` block configures Promtail's behavior as an HTTP server:
...
 
-The `clients` block configures how Promtail connects to instances of
-Loki:
+`client_config` configures how Promtail connects to instances of Loki:
```
Comment (trevorwhitney) [inline docs/sources/clients/promtail/configuration.md:162]: I'm not sure this is correct.

The top level config keys appear to be `client` and `clients` (https://github.com/grafana/loki/blob/51fbc95bbd27b9edc584c66fd7d18256e11ec33a/clients/pkg/promtail/config/config.go#L30), so we shouldn't change that here.

**Review by trevorwhitney — CHANGES_REQUESTED** [review]
I'm not sure this change is correct, and it breaks the pattern we have for the other sections, WDYT?

File: docs/sources/clients/promtail/configuration.md (hunk 1 above)
Comment (DylanGuedes) [reply docs/sources/clients/promtail/configuration.md:160]: Omg you right :facepalm: I got so used to the old syntax that I forgot to look at the other sections for comparison. Thank you!

File: docs/sources/clients/promtail/configuration.md (hunk 2 above)
Comment (DylanGuedes) [reply docs/sources/clients/promtail/configuration.md:162]: True, my suggestion is misleading. My intent was actually to say that the `<client_config>` type configures Promtail connection to Loki.

**Comment by DylanGuedes** [comment]
> I'm not sure this change is correct, and it breaks the pattern we have for the other sections, WDYT?

You're 100% right :smile: believe I was having a brain fog when I created this lool. Closing...

**Pull request closed without merging**